products/va-mobile-app/platform/risks-of-skipping-staging-review.md

```diff
@@ -0,0 +1,124 @@
+# 202505 Risks of Skipping Staging Review for the VA Mobile App: Impact on QA, Accessibility, 
+Content, Information Architecture, and Design 
+
+## Risks of Skipping Staging Review 
+
+When teams skip the Staging Review, they lose: 
+
+- The last chance to confirm the product meets VA Experience Standards before launch. This 
+step is essential for ensuring compliance and consistency with the VA Design System, as well 
+as adherence to recognized accessibility guidelines for apps. An independent review from 
+the Governance Team, who often find issues the development teams miss. Without this review, 
+teams develop "tunnel vision" and overlook problems. 
+- The chance to catch critical, launch-blocking issues early, such as low test coverage, 
+inaccessible content, missing navigation, or confusing user flows. 
+- A clear process to create and track GitHub issues for all findings, which helps teams 
+follow up and close out problems. 
+
+While the mobile team will do QA reviews to catch issues, Governance adds extra value. Governance 
+checks how the product fits into the overall VA digital experience and adds more review layers. 
+While no team can catch every issue alone, working together helps us find more problems before launch. 
+Skipping Staging Review makes it more likely that teams will launch the app with hidden bugs, 
+accessibility barriers, or compliance failures. Teams also miss objective feedback and a way to 
+flag launch-blocking issues. These risks directly impact Veterans who rely on the VA Mobile App. 
+
+### Practice Area Risks 
+
+The next sections outline the risks of skipping Staging Review for the VA Mobile App. These risks 
+affect Veterans and impact how the VA Mobile App is developed and maintained. 
+
+#### Quality Assurance QA Risks 
+
+The VA Mobile App team tests new features, but skipping Staging Review removes a key checkpoint 
+for thorough testing and monitoring. Without this review, itʼs harder to prove that the team met 
+QA standards or is ready for incidents after launch. 
+
+Risks of skipping Staging Review for QA include: 
+
+- Low test coverage (below 80%, which leaves code untested. 
```

Comment:
Active risk - addressable

@TODO
1. Increase our test coverage to 80%
2. Add CI check to fail PRs that fall below this threshold

Reply:
Ticket to evaluate test coverage: https://github.com/department-of-veterans-affairs/va-mobile-app/issues/11064